### PR TITLE
Buffs MP and Officer Armor's Bullet Armour

### PR DESCRIPTION
**Pull request by MistChristmas**
# About the pull request
Small simple change, some additional clean
MPs and derived Officer Armour now gets Medium Bullet Armour (20) Instead of Low (10).
Also removed some redundant armour variables that clarified it was the same level already being derived from.
Also moved icon_state of Warden and Chief MP around to be in the same order as most others.

# Explain why it's good for the game
MP and Officer Armor is already limited and restricted to certain roles, it shouldn't be worse than standard.
And MPs gear should lean toward human threat rather than anything else.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: MistChristmas
balance: Buffed MP and Officer Armor's Bullet Armour.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/marine_armor/_marine_armor.dm [hunk 1]
```diff
@@ -254,7 +254,7 @@
 	desc = "A standard Colonial Marines M2 Pattern Chestplate. Protects the chest from ballistic rounds, bladed objects and accidents. It has a small leather pouch strapped to it for limited storage."
 	icon_state = "mp_armor"
 	armor_melee = CLOTHING_ARMOR_MEDIUMHIGH
-	armor_bullet = CLOTHING_ARMOR_LOW
+	armor_bullet = CLOTHING_ARMOR_MEDIUMHIGH
```
Comment (Diegoflores31) [inline code/modules/clothing/suits/marine_armor/_marine_armor.dm:257]: this is an insane jump on armor values . at that point why use riot armor if the base just has 5 less ??? 
i think MEDIUMLOW or MEDIUM would be better

Comment (Diegoflores31) [reply code/modules/clothing/suits/marine_armor/_marine_armor.dm:257]: remember this armor has MEDIUMHIGH MELEE and very little slowdown

Comment (MistChristmas) [reply code/modules/clothing/suits/marine_armor/_marine_armor.dm:257]: I'll amend it to medium. Unless you want to split MP and Officer armor.

File: code/modules/clothing/suits/marine_armor/_marine_armor.dm [hunk 2]
```diff
@@ -309,7 +307,7 @@
 	name = "\improper M3 pattern general officer armor"
 	desc = "A well-crafted suit of M3 Pattern Armor with a gold shine. It looks very expensive, but shockingly fairly easy to carry and wear."
 	icon_state = "general"
-	armor_bullet = CLOTHING_ARMOR_MEDIUMHIGH
+	armor_bullet = CLOTHING_ARMOR_HIGH
```
Comment (Drulikar) [inline code/modules/clothing/suits/marine_armor/_marine_armor.dm:310]: This doesn't need to change; what gave you the notion this armor is worse than base? It already has 5 more than marine armor; you're just making it have 10 more than marine armor.

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/marine_armor/_marine_armor.dm (hunk 2 above)
Comment (MistChristmas) [reply code/modules/clothing/suits/marine_armor/_marine_armor.dm:310]: I presume this is event Armor for a High Command visitor and thought it probably should be on par with the CO. 
I can drop it back down.

**Comment by Drulikar** [comment]
Undraft if you're happy with the pr

**Comment by MistChristmas** [comment]
> Undraft if you're happy with the pr

Okie doke.

**Review by Drulikar — APPROVED** [review]

**Pull request merged**